README.md

```diff
@@ -38,8 +38,19 @@ Quick start (AMD64 linux) -- install wireguard, then, after making sure `/usr/lo
 
 Copy the generated configuration file to your device and connect!
 
-To send configurations, ffsend (with separately transferred password) or a
-local QR code generator may be used.
+To send configurations, here are a few suggestions.
+- [ffsend](https://github.com/timvisee/ffsend), the most straightforward option;
+- [magic wormhole](https://magic-wormhole.readthedocs.io/), a more advanced
+  option, where the file never passes through another server;
+- [womroleh-william](https://github.com/psanford/wormhole-william), a Go
+  implementation of the above.
+
+For the above options, one should transfer the password separately.
+
+A local QR code generator, such as the popular
+[qrencode](https://fukuchi.org/works/qrencode/) may also be used to generate a
+QR code of the configuration. For instance, from the example above:
+`qrencode -t ansiutf8 < dsnet-banana.conf`
```

Comment:
oh, nice. Note that we could pipe directly `dsnet add | qrencode -t ansiutf8` -- this is more secure as the config file does not touch the disk. It works because the prompts that dsnet gives are on STDERR so they won't be qrencoded.